### DISCUSSION TITLE
Are custom steps possible with the maven plugin?

### DISCUSSION THREAD

**Discussion by davkuyek** [diffplug/spotless#1953] [Ideas]
Hello! Thank you for producing such a high quality plugin.

I am interested in using a custom step using the Maven plugin but I noticed the Maven equivalent to the Gradle section of the plugin Readme.md does not discuss custom steps. Is it currently possible to create and apply custom steps with the Maven plugin?

**Comment by nedtwigg** [comment]
You can do regex replace, but that's it. A PR adding support for custom steps in maven would be very welcome.
